### PR TITLE
IDVA3-1617 Feature/add tests to index

### PR DESCRIPTION
**Pull request by JasSmith-ch**
- added tests to formatDateBorn
- also added a test for internationaliseDate

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: test/routers/utils/index.unit.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+});
+
+describe("internationaliseDate tests", () => {
+    test("formats date in en-GB when lang is en", () => {
+        const date = "2024-06-06";
+        const lang = "en";
+        const formattedDate = internationaliseDate(date, lang);
+        expect(formattedDate).toBe("6 June 2024");
+    });
+});
```
Comment (PhilipGlover) [inline test/routers/utils/index.unit.ts:34]: could we add a few welsh versions of these tests?

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: test/routers/utils/index.unit.ts (hunk 1 above)
Comment (PhilipGlover) [reply test/routers/utils/index.unit.ts:48]: welsh added

File: test/routers/utils/index.unit.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        expect(formattedDate).toBe("6 June 2024");
+    });
+
+    it("formats date in Welsh when lang is cy", () => {
+        const date = "1995-06-06";
+        const lang = "cy";
+        const formattedDate = internationaliseDate(date, lang);
+        expect(formattedDate).toBe("6 Mehefin 1995");
+    });
+});
```
Comment (PhilipGlover) [inline test/routers/utils/index.unit.ts:48]: should have noted before, but the coverage report shows 75 %
Screenshot 2024-06-12 at 7 56 54 AM
- adding tests for invalid date in `internationaliseDate` will bring this up
Screenshot 2024-06-12 at 7 55 07 AM

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by PhilipGlover — APPROVED** [review]
comments addressed, looks good

**Pull request merged**